src/SDKConnector.php

```diff
@@ -273,7 +274,11 @@ public function testConnection(KeyInterface $key = NULL) {
     try {
       // We use the original, non-decorated organization controller here.
       $oc = new OrganizationController($client);
-      $oc->load($credentials->getKeyType()->getOrganization($credentials->getKey()));
+      /* @var \Apigee\Edge\Api\Management\Entity\Organization $org */
+      $org = $oc->load($credentials->getKeyType()->getOrganization($credentials->getKey()));
+      if (empty($org->id())) {
+        throw new ApiException('Failed to load a valid organization.');
```

Comment:
ApiExceptions are thrown by the API client and not by the module. This is an invalid use of that exception here. Maybe let's use the module-specific InvalidArgumentException for this instead https://github.com/apigee/apigee-edge-drupal/blob/8.x-1.x/src/Exception/InvalidArgumentException.php

Reply:
Sounds good, updated code to use `InvalidArgumentException`.